CONTRIBUTING.rst

```diff
@@ -4,65 +4,42 @@ Releasing
 Summary
 -------
 
-- Create a release branch from the develop branch in the original ``epcpower/sunspec-demo`` repository, not in a fork.
-- Create a commit for a release candidate inside the release branch.
-- Create a Pull Request for the release. The same branch and PR will be used for both the release candidates and  the final release.
-- Get an approving review.
-- Once approved, tag the last commit in the branch as a release candidate.
-- Push the tag. This will trigger a build including the upload of artifacts.
-- Prepare a commit for a final release and push it to the release branch.
-- Get an approving review for the final release.
-- Tag the commit using the final release version. This will trigger the upload of artifacts.
-- Merge the release branch.
+- The sunspec-demo repository uses the `feature branch workflow <https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow>`_ process to control development.
+- The ``main`` branch must never be broken.
+- Development is performed in feature branches, requiring peer reviewed pull requests to merge into ``main``.
+- A release also require a branch, detailed in the step-by-step instructions below.
+- The ``versioneer`` package automatically manages version handling code.
 
 
 Step-by-step
 ------------
 
 - Define the final release version you are preparing.
 
-  - ``epcpower/sunspec-demo`` uses `CalVer <https://calver.org/>`_ of the form ``YYYY.0M.micro`` with the micro version just incrementing.
+  - ``epcpower/sunspec-demo`` uses `SemVer <https://semver.org/>`_ of the form ``MAJOR.MINOR.PATCH``.
   - Normalize the version according to `PEP 440 <https://www.python.org/dev/peps/pep-0440/#normalization>`_.
 
-- Create a release branch with a name of the form ``release-v2021.09`` starting from the ``develop`` branch.
+- Create a release branch with a name of the form ``release-vMAJOR.MINOR.PATCH`` (example: ``release-v1.2.4``) starting from the ``main`` branch.
 
-  - On the new release branch you will commit all tagged release candidate commits as well as the final tagged release commit.
+- Update the ``HISTORY.rst`` file.
 
-- Update the version to the release candidate with the first being ``rc1`` (as opposed to 0).
+  - Summarize all of the features and bug fixes since the previous release.
```

Comment:
I think that, in general, the history needs to be maintained along the way.  Each PR ought to, if needed, document its change.  Certainly some changes don't require a user-facing description in the history.